src/kvilib/ext/KviCryptEngine.cpp

```diff
@@ -157,7 +157,7 @@
 	KviCryptEngine::KviCryptEngine()
 	{
 #ifdef COMPILE_CRYPT_SUPPORT
-		setLastError(__tr2qs("Invalid crypt engine"));
+		setLastError(__tr2qs("Invalid encryption engine"));
```

Comment:
Aha! It does seem like `encryption engine` is right. Perhaps replace the above `encrypting engines` with `encryption engines`?